themes-default/slim/src/components/show-header.vue

```diff
@@ -21,24 +21,24 @@
                         >{{ show.title }}</app-link> / Season {{ season }}<template v-if="episode && manualSearchType !== 'season'"> Episode {{ episode }}</template>
                     </span>
                 </div>
-                <div v-if="type !== 'snatch-selection' && show.seasons && show.seasons.length >= 1" id="show-specials-and-seasons" class="pull-right">
-                    <span class="h2footer display-specials" v-if="show.seasons.find(season => ({ season }) => season === 0)">
+                <div v-if="type !== 'snatch-selection' && seasons.length > 1" id="show-specials-and-seasons" class="pull-right">
```

Comment:
@p0psicles 
Not sure if this was done intentionally - you changed `seasonJump` to only show up when there are at least 2 seasons (before it was at least 1 season).

I guess it makes sense (you don't need to jump to season if there's only one), but I think it should be consistent - if there are any seasons, we should have the jump to season (and show/hide specials)

Reply:
I didn't intend to change behavior